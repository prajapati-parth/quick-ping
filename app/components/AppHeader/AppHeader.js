import React, {Component} from 'react'
import {
	Text,
	View,
    StyleSheet,
    ToolbarAndroid
} from 'react-native'
import GlobStyles from '../../config/styles'
import GlobStrings from '../../config/strings'

export default class AppHeader extends Component {
    render() {
        return (
            <View>
                <ToolbarAndroid
                    title="quickping"
                    style={styles.headerStyle} />
            </View>
        )
    }
}

const styles = StyleSheet.create({
    headerStyle: {
        height: 50,
        backgroundColor: '#F44336'
    }
})