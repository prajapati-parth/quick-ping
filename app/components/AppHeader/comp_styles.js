import {StyleSheet} from 'react-native'
import GlobStyles from '../../config/styles'

const CompStyles = StyleSheet.create({
    headerContainer: {
        height: GlobStyles.toolbarHeight,
        backgroundColor: GlobStyles.primary,
        padding: GlobStyles.boundaryPadding,
        justifyContent: 'center',
        alignItems: 'center',
        elevation: GlobStyles.elevationForShadow
    },
    headerText: {
        color: '#FFFFFF',
        fontSize: GlobStyles.titleFontSize
    }
})

export default CompStyles