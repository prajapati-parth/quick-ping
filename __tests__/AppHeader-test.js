import 'react-native'
import React from 'react'
import AppHeader from '../app/components/AppHeader'

import renderer from 'react-test-renderer'

test('renders correctly', () => {
    const tree = renderer.create(
        <AppHeader />
    ).toJSON()
    expect(tree).toMatchSnapshot()
})