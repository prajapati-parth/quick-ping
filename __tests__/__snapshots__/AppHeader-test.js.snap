// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F44336",
      "elevation": 5,
      "height": 36,
      "justifyContent": "center",
      "padding": 10,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 16,
      }
    }
  />
</View>
`;
